spot_wrapper/wrapper.py

```diff
@@ -436,15 +430,19 @@ def __init__(
         self._state = RobotState()
         self._trajectory_status_unknown = False
         self._command_data = RobotCommandData()
+        self._is_licensed_for_choreography = False
 
         try:
             self._sdk = create_standard_sdk(SPOT_CLIENT_NAME)
         except Exception as e:
             self._logger.error("Error creating SDK object: %s", e)
             self._valid = False
             return
-        if HAVE_CHOREOGRAPHY:
+        try:
             self._sdk.register_service_client(ChoreographyClient)
```

Comment:
@ksharma-bdai we shouldn't merge this code until we test on a robot that doesn't have the choreography service.

Reply:
Functionality confirmed on robot without a license